code/datums/station_traits/negative_traits.dm

```diff
@@ -0,0 +1,91 @@
+/datum/station_trait/carp_infestation
+	name = "Carp infestation"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Dangerous fauna is present in the area of this station"
+	trait_to_give = STATION_TRAIT_CARP_INFESTATION
+
+/datum/station_trait/distant_supply_lines
+	name = "Distant supply lines"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 3
+	show_in_report = TRUE
+	report_message = "Due to the distance to our normal supply lines, cargo orders are more expensive."
+	blacklist = list(/datum/station_trait/strong_supply_lines)
+
+/datum/station_trait/distant_supply_lines/on_round_start()
+	SSeconomy.pack_price_modifier *= 1.2
+
+/datum/station_trait/late_arrivals
+	name = "Late Arrivals"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 2
+	show_in_report = TRUE
+	report_message = "Sorry for that, we didn't expect to fly into that vomiting goose while bringing you to your new station."
+	trait_to_give = STATION_TRAIT_LATE_ARRIVALS
+	blacklist = list(/datum/station_trait/random_spawns)
+
+/datum/station_trait/random_spawns
+	name = "Drive-by landing"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 2
+	show_in_report = TRUE
+	report_message = "Sorry for that, we missed your station by a few miles, so we just launched you towards your station in pods. Hope you don't mind!"
+	trait_to_give = STATION_TRAIT_RANDOM_ARRIVALS
+	blacklist = list(/datum/station_trait/late_arrivals)
+
+/datum/station_trait/blackout
+	name = "Blackout"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Station lights seem to be damaged, be safe when starting your shift today."
+
+/datum/station_trait/blackout/on_round_start()
+	. = ..()
+	for(var/a in GLOB.apcs_list)
+		var/obj/machinery/power/apc/current_apc = a
+		if(prob(30))
+			current_apc.overload_lighting()
+
+/datum/station_trait/empty_maint
+	name = "Cleaned out maintenance"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Our workers cleaned out most of the junk in the maintenace areas."
+	blacklist = list(/datum/station_trait/filled_maint)
+	trait_to_give = STATION_TRAIT_EMPTY_MAINT
+
+
+/datum/station_trait/overflow_job_bureacracy
+	name = "Overflow bureacracy mistake"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 5
+	show_in_report = TRUE
+	var/list/jobs_to_use = list("Clown", "Bartender", "Cook", "Botanist", "Cargo Technician", "Mime", "Janitor", "Prisoner")
+	var/chosen_job
+
+/datum/station_trait/overflow_job_bureacracy/New()
+	. = ..()
+	chosen_job = pick(jobs_to_use)
+
+/datum/station_trait/overflow_job_bureacracy/get_report()
+	return "[name] - It seems for some reason we put out the wrong job-listing for the overflow role this shift...I hope you like [chosen_job]s."
+
+/datum/station_trait/overflow_job_bureacracy/on_round_start()
+	. = ..()
+	SSjob.set_overflow_role(chosen_job)
+
+/datum/station_trait/slow_shuttle
```

Comment:
Undocumented feature? Same for the fast shuttle trait

Reply:
ty